docs/src/modules/javascript/pages/views.adoc

```diff
@@ -147,31 +128,30 @@ include::example$js-customer-registry/customer-event-sourced-view.js[tag=process
 
 <1> Each update method in the Protobuf definition should have a corresponding JavaScript function in `view.setUpdateHandlers`.
 
-The first function parameter should correspond to the parameter in the Protobuf service call, that is, the event.
-
-A second parameter can optionally be defined for the previous state. For the first event of an Event Sourced Entity or for the first change of a Value Entity there is no previous state and `null` is used for the state parameter.
-
-The function may also take a `UpdateHandlerContext` parameter.
+The first function parameter should correspond to the parameter in the Protobuf service call, that is, the event. You can optionally define a second parameter for the previous state. For the first event of an Event Sourced Entity or for the first change of a Value Entity there is no previous state and `null` is used for the state parameter. The function may also take a `UpdateHandlerContext` parameter.
 
-NOTE: Events from an Event Sourced Entity is the canonical use case for this kind of update transformation, but it can also be used for Value Entities. For example, if the View representation is different from the Entity state.
+NOTE: This type of update transformation is a natural fit for Events emitted by an Event Sourced Entity, but it can also be used for Value Entities. For example, if the View representation is different from the Entity state you might want to transform it before presenting the View to the client.
 
-=== Registering
+[#ES_register]
+=== Register the View
 
-Register the View in `AkkaServerless`:
+In the implementation, register the View with `AkkaServerless`:
 
-.index.js
+.customer-event-sourced-view.js
 [source,js,indent=0]
 ----
-include::example$js-customer-registry/index.js[tag=register-event-sourced]
+include::example$js-customer-registry/customer-event-sourced-view.js[tag=register]
 ----
 
-.customer-event-sourced-view.js
+In the client, use the `addComponent` function to register the View. For example:
```

Comment:
`In the client` doesn't make sense here. It's registering with the user function / service / server.

Reply:
same change as above.

Reply:
Not actually changed in this PR

Reply:
See https://github.com/lightbend/akkaserverless-javascript-sdk/pull/139/files